pulp_docker/tests/functional/api/test_download_content.py

```diff
@@ -86,18 +85,27 @@ def test_all(self):
         distribution = client.post(DISTRIBUTION_PATH, body)
         self.addCleanup(client.delete, distribution['_href'])
 
-        # Pick a content unit, and download it from both Pulp Fixtures…
-        unit_path = choice(get_docker_image_paths(repo))
-        fixtures_hash = hashlib.sha256(
-            utils.http_get(urljoin(DOCKER_FIXTURE_URL, unit_path))
-        ).hexdigest()
+        # FIXME: Figure out how to download content units
```

Comment:
I need help to figure out how to get the blobsums from Pulp to compare with remote
Or how to make an `internal pull` of the image without https.